packages/stark-core/src/validation/validators/is-bic/is-bic.validator.fn.ts

```diff
@@ -1,6 +1,11 @@
+/**
+ * @ignore
+ * Name of the validator, in case injection is needed.
+ */
 export const starkIsBICValidatorName: string = "starkIsBIC";
 
 /**
+ * @ignore
```

Comment:
Maybe we can move or copy this block to the place where this function is documented, cause this info seems to be important to undertand how this validation works